ci/download_houdini2.sh

```diff
@@ -9,11 +9,11 @@ HOUDINI_SECRET_KEY="$5"
 
 if [ "$HOUDINI_CLIENT_ID" == "" ]; then
     echo "HOUDINI_CLIENT_ID GitHub Action Secret needs to be set to install Houdini builds"
-    exit 1
+    exit 0
```

Comment:
This script is only run in the weekly Houdini workflow. Note that this change has been primarily made for repos that are forks of OpenVDB. Right now, if you fork OpenVDB and create a fork/master that mirrors the parent/master, you'll introduce a recurring GitHub Action workflow once a week that fails unless you add the secret keys or otherwise disable GitHub Actions. This makes it silently fail if it doesn't find the secret key.